components/teams-service/storage/storage.go

```diff
@@ -6,32 +6,21 @@ import (
 	"time"
 
 	"github.com/chef/automate/lib/stringutils"
-	uuid "github.com/chef/automate/lib/uuid4"
 )
 
-// WARNING
-// TODO (tc): The storage interface is still using V1 verbiage, so
-// name is really the ID in V2 terms. We'll refactor at GA when V1 is removed.
-//
 // Storage is the interface provided by our various storage backends.
 type Storage interface {
-	DeleteTeam(context.Context, uuid.UUID) (Team, error)
-	EditTeam(context.Context, Team) (Team, error)
 	GetTeams(context.Context) ([]Team, error)
-	GetTeam(context.Context, uuid.UUID) (Team, error)
-	StoreTeam(ctx context.Context, name string, description string) (Team, error)
-	StoreTeamWithProjects(ctx context.Context, name string, description string, projects []string) (Team, error)
-	RemoveUsers(ctx context.Context, id uuid.UUID, users []string) (Team, error)
-	GetTeamByName(ctx context.Context, teamName string) (Team, error)
-	PurgeUserMembership(ctx context.Context, userID string) (teamsUpdated []uuid.UUID, err error)
-	AddUsers(ctx context.Context, teamID uuid.UUID, users []string) (Team, error)
+	StoreTeam(ctx context.Context, id string, name string, projects []string) (Team, error)
```

Comment:
`StoreTeam` was the v1 function, but I decided to delete it and then rename `StoreTeamWithProjects` to `StoreTeam` to preserve shorter function names. (same with GetTeamByName, DeleteTeamByName, EditTeamByName)